applications/nrf5340_audio/sysbuild/mcuboot/prj.conf

```diff
@@ -7,9 +7,11 @@
 CONFIG_SIZE_OPTIMIZATIONS=y
 CONFIG_MAIN_STACK_SIZE=20480
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
-CONFIG_PCD_APP=y
 
 # Enable serial debug for MCUBOOT
 CONFIG_CBPRINTF_NANO=y
 CONFIG_LOG=y
 CONFIG_UART_NRFX=y
+
+CONFIG_PCD_APP=y
+CONFIG_BOOT_UPGRADE_ONLY=y
```

Comment:
Why is this here?

Reply:
It shouldn't :) I forgot to move it before push, will move to `overlay-mcuboot-ext-dfu.conf` so they are only applied when  external flash DFU is enabled